src/Page/Guide/View.elm

```diff
@@ -35,7 +39,7 @@ view language guide allGuides allStories =
                   , viewPrintGuide language
                   , viewRelatedGuideTeasers language guide.relatedGuideList allGuides
                   ]
-                , [ viewRelatedStoryTeasers language guide.relatedStoryList allStories ]
+                , [ viewRelatedStoryTeasers language guide.relatedStoryList allStories, Page.Shared.View.viewCallForStory language call ]
```

Comment:
might be easier to read on 2 lines

Reply:
I've changed it, I've generally been trusting the formatter to make these decisions.